tests/test_integration/test_code_generation.py

```diff
@@ -361,7 +352,10 @@ def stencil(
             out_field = in_field * parameter
 
 
-@pytest.mark.parametrize("backend", OLD_BACKENDS)
+# TODO: Enable variable offsets on gtc:dace backend
+@pytest.mark.parametrize(
+    "backend", [backend for backend in ALL_BACKENDS if "dace" not in backend.values[0]]
+)
 def test_variable_offsets(backend):
```

Comment:
Variable offsets work on all the new backends now (this test should have included them) with the exception of `gtc:dace`, so I made a TODO.